src/us/kbase/workspace/test/workspace/WorkspaceTest.java

```diff
@@ -7565,16 +7657,9 @@ public void getObjectsMixedCalls() throws Exception {
 			destroyGetObjectsResources(lwod);
 		}
 		// test getting info only
-		final List<ObjectInformation> loi = ws.getObjectInformation(user1, Arrays.asList(
-				leaf2oi,
-				simplerefoi,
-				(ObjectIdentifier) new ObjectIDWithRefPath(leaf1oi), // auto lookup
-				(ObjectIdentifier) new ObjectIDWithRefPath(simplerefoi, Arrays.asList(leaf1oi)),
-				(ObjectIdentifier) new ObjectIDWithRefPath(simplerefoi, null),
```

Comment:
You'll notice that this entry is missing from the `nodata` list. This is because this constructor call produces, effectively, the same ObjectIdentifier as as `simplerefoi`, other than the class. With the builder even that distinction is gone.